dev/prod_builders.json

```diff
@@ -1149,7 +1149,7 @@
     {
       "name": "Mac_ios platform_channels_benchmarks",
       "repo": "flutter",
-      "task_name": "mac_ios_platform_channels_benchmarks",
+      "task_name": "mac_ios_platform_channels_benchmarks_ios",
```

Comment:
I guess the task name also needs to be updated here: https://devops-console-oss.corp.google.com/flutter/infra/+/master:config/devicelab_config.star;l=355, which controls which test file to run.

Reply:
Done here: https://github.com/flutter/infra/pull/446  Let me know what order you'd like me to merge if you have a preference.  Thanks again @keyonghan

Reply:
Sure. After the infra PR lands, we can land this PR with updated builder name and task name.